components/badge/badge.css

```diff
@@ -0,0 +1,42 @@
+.badge-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+  }
+  .badge {
+    display: flex;
+    gap: 30px;
```

Comment:
Try not using absolute units instead use responsive units(rem or ems)

Reply:
Sure,I'm changing it right away